language/go/fileinfo.go

```diff
@@ -307,6 +310,9 @@ func goFileInfo(path, srcdir string) fileInfo {
 					info.hasMainFunction = true
 					break
 				}
+				if info.isTest {
+					info.numParallel += countParallel(fdecl)
```

Comment:
Should we really be summing these up? My understanding has been that individual test functions in a file are not run in parallel, only the `t.Parallel` funcs in them. In that case taking the maximum would be more appropriate.